### PR TITLE
Fixes #27206 - improve mysql migration

### PR DESCRIPTION
**Pull request by lzap**
[no ci]
[skip ci]

Do not merge yet. This PR will contain various small improvements we can
make for the prod2dev rake task for smoother MySQL to Postgres
migration.

### CODE REVIEW

**Comment by lzap** [comment]
@tbrisker corrected rescue block, new task. I am gonna rebase https://github.com/theforeman/foreman/pull/6974 on top of this

File: lib/tasks/convert.rake [hunk 1]
```diff
@@ -81,77 +81,84 @@ namespace :db do
...
 
-        print "#{count} records converted in #{Process.clock_gettime(Process::CLOCK_MONOTONIC) - time} seconds\n"
+          print "#{count} records converted in #{Process.clock_gettime(Process::CLOCK_MONOTONIC) - time} seconds\n"
+        rescue StandardError => e
+          print "Unable to convert #{table_name}, skipping: #{e}"
+        end
       end
     end
+
+    desc 'Migrate MySQL (prod) to PostgreSQL (dev).   DANGER Deletes all data in the development database.   Assumes both schemas are already migrated.'
```
Comment (tbrisker) [inline lib/tasks/convert.rake]:
```suggestion
    desc 'Migrate MySQL (prod) to PostgreSQL (dev).   DANGER Deletes all data in the development database.'
```

**Comment by lzap** [comment]
Rebased, also removed those confusing four spaces, it was misformatted when `-T` was called.

**Review by tbrisker — APPROVED** [review]
Thanks @lzap !

**Comment by tbrisker** [comment]
1.23 - d2da0214b

**Pull request merged**